content/F2016/F16-event-gooutside.md

```diff
@@ -0,0 +1,20 @@
+title: WiCS and CSC Go Outside
+Date: 2016-10-13 18:00
+Category: Events
+Tags: social
+Slug: go-outside-s16 
+Author: Shivangi Patwardhan 
+Summary: We're having a bonfire with the CSC. S'mores and a bonfire!
+
+WiCS and CSC are super excited for this upcoming event! We hope that 
+you all can make it and bring your friends!
+
+There will be free s'mores and light dinner will be served :)
+
+## Event Details ##
+
++ **Who:** Everyone
++ **What:** Bonfire!
++ **Where:** Columbia Lake 2 Firepit
++ **When:** Thursday October 13 6:00
```

Comment:
It looks weird to me without commas? Idk this is mostly a personal preference. What do you think of Thursday, October 13, 6:00? In any case, you should change it to 6:00 pm.